src/Native/System.Native/pal_networking.cpp

```diff
@@ -21,16 +21,14 @@ static_assert(PAL_TRY_AGAIN == TRY_AGAIN, "");
 static_assert(PAL_NO_RECOVERY == NO_RECOVERY, "");
 static_assert(PAL_NO_DATA == NO_DATA, "");
 static_assert(PAL_NO_ADDRESS == NO_ADDRESS, "");
-static_assert(sizeof(uint8_t) == sizeof(char), ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
+static_assert(sizeof(uint8_t) == sizeof(char),
+              ""); // We make casts from uint8_t to char for OS functions, make sure it's legal
```

Comment:
I'm trying to keep this pack from having any code changes at all, so if we need to remove the check then that should be in a different pack...I don't want this to have any side effects. I can shorten the comment to keep it on one line, but I would really prefer to keep the check and not remove it in this pack